### PR TITLE
Feature/ersc/rdphoen 1314 disable serializer pwm

### PR DESCRIPTION
**Pull request by iris-ersc**
Disable PWM clock for serializer

~~Depends on: https://bitbucket.iris-sensing.net/projects/COMP/repos/hal_imager/pull-requests/109/overview~~ (merged)

### CODE REVIEW

**Comment by iris-ersc** [comment]
Pipeline: https://gitlab.devops.defra01.iris-sensing.net/MIRROR/iris-kas/-/pipelines/1960

**Review by jan-hannig — APPROVED** [review]

**Review by linkjumper — APPROVED** [review]
Tested!

File: recipes-kernel/linux/linux-fslc-iris/0038-RDPHOEN-1314-Disable-PWM-clock-for-serializer.patch [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+diff --git a/drivers/media/i2c/tc358746.c b/drivers/media/i2c/tc358746.c
+index f5c30a67d321..0cd7e8073952 100644
+--- a/drivers/media/i2c/tc358746.c
++++ b/drivers/media/i2c/tc358746.c
+@@ -1595,7 +1595,7 @@ static int tc358746_probe_fw(struct tc358746_state *state)
+ 	 * The PLL input clock is obtained by dividing refclk by pll_prd.
+ 	 * It must be between 4 MHz and 40 MHz, lower frequency is better.
+ 	 */
+-	pll_prediv = DIV_ROUND_CLOSEST(refclk, 4000000);
++	pll_prediv = 1; // don't divide anything ... very hacky!!
```
Comment (Eximmius) [inline recipes-kernel/linux/linux-fslc-iris/0038-RDPHOEN-1314-Disable-PWM-clock-for-serializer.patch:49]: DIV_ROUND_DOWN_ULL maybe?

**Review by Eximmius — CHANGES_REQUESTED** [review]

File: recipes-kernel/linux/linux-fslc-iris/0038-RDPHOEN-1314-Disable-PWM-clock-for-serializer.patch (hunk 1 above)
Comment (Eximmius) [reply recipes-kernel/linux/linux-fslc-iris/0038-RDPHOEN-1314-Disable-PWM-clock-for-serializer.patch:49]: Not available for our current linux, maybe a custom div_round_down?

**Review by ThomasWNoack — CHANGES_REQUESTED** [review]
Please rebase manually

**Review by Eximmius — APPROVED** [review]

**Review by ThomasWNoack — APPROVED** [review]

**Comment by ThomasWNoack** [comment]
## Maintainer Checklist
- Is there a changelog entry? *Yes*
- Do the commit messages comply with the specifications? *Yes*
- Was a Jenkins run performed and was it error-free? *Yes*
- Is a manual rebase necessary? *no*

**Pull request merged**